plugin/src/test/kotlin/org/neotech/plugin/rootcoverage/IntegrationTest.kt

```diff
@@ -102,9 +102,9 @@ class IntegrationTest(
                 executeGradleTasks(listOf("clean", "nexusoneapi30DebugAndroidTest", "nexusoneapi30DemoDebugAndroidTest"))
             }
 
-            listOf("coverageReport", "rootCodeCoverageReport", "--stacktrace")
+            listOf("coverageReport", "rootCodeCoverageReport", "--stacktrace", "--rerun-tasks")
```

Comment:
Could you elaborate on why you added this? Did you run into any issues with tests not working correctly if everything was up-to-date?

Reply:
3 week ago, when I was working on this PR, during local execution of integration tests, at some time I started to receive errors connected to gradle tasks's statuses. As I remember, the issue was that tasks had `UP_TO_DATE` status, instead of `SUCCESS`. So, I added `--rerun-tasks` parameter in order to force tasks execution.

However, today I am not able to reproduce this issue during local execution of integration tests without `--rerun-tasks`. I think it was some kind of gradle cache issue. So, I guess we could just remove `--rerun-tasks` until a stable reproduction scenario emerges.